angular-docker-sample-app/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '1'
```

Comment:
On running `docker-compose build`, I am getting this:

```bash
$ docker-compose build
WARN[0000] /home/lakshaya/RAndD/angular-docker-sample/angular-docker-sample-app/docker-compose.yml: `version` is obsolete
```

Please check if we can omit this.

Reply:
Removed this line, docker works fine, please check once